maintainers/maintainer-list.nix

```diff
@@ -14842,4 +14842,10 @@
     github = "yisuidenghua";
     githubId = 102890144;
   };
+  water-sucks = {
```

Comment:
Sorry to be nitpicky, though there are some entries ignoring the request, it will be nice to keep the list alphabetically sorted.

https://github.com/NixOS/nixpkgs/blob/f0ec9e1f968bbeb214dd0c92989dc8a8beb7a724/maintainers/maintainer-list.nix#L49